### DISCUSSION TITLE
Kitty does not switch to other applications

### DISCUSSION THREAD

**Discussion by dudicoco** [kovidgoyal/kitty#7318] [Q&A] [answered]
Hey, wanted to ask in the discussions instead of reporting a bug since i'm not sure if this is a problem with my machine or with kitty.

When opening a file/url with external programs (for example `code example.txt`), the file/url opens in the external program but it does not switch to the external program automatically (I have to do it manually).

In the past it did switch automatically so I'm not sure why it's misbehaving now (could be related to MacOS version?)

**Comment by kovidgoyal** [comment, marked as answer]
You turned on secure text entry turn it off again.

**Reply by dudicoco** [reply]
thanks, that was it. Is there a way to disable all of these default keybindings in order to avoid such mistakes in the future?

**Reply by dudicoco** [reply]
I guess this is what i'm looking for https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.clear_all_shortcuts

**Reply by dudicoco** [reply]
that didn't help, any suggestions @kovidgoyal?

**Reply by dudicoco** [reply]
sorry, it does work, i didn't restart kitty properly
